test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -408,140 +406,155 @@ public void orderActivitieFiltersAndSorts() {
-            "'type':'SchedulePlan'"+
-        "}");
-        
-        Map<String,DateTime> events = Maps.newHashMap();
-        events.put("enrollment", DateTime.now().withZone(DateTimeZone.UTC).minusDays(3));
-        when(activityEventService.getActivityEventMap("AAA")).thenReturn(events);
-        
-        ClientInfo info = ClientInfo.fromUserAgentCache("Parkinson-QA/36 (iPhone 5S; iPhone OS/9.2.1) BridgeSDK/7");
-        
-        SchedulePlan voiceActivityPlan = BridgeObjectMapper.get().readValue(json, SchedulePlan.class);
-        List<SchedulePlan> schedulePlans = Lists.newArrayList(voiceActivityPlan);
-        when(schedulePlanService.getSchedulePlans(info, new StudyIdentifierImpl("test-study"))).thenReturn(schedulePlans);
-        
-        ScheduleContext context = new ScheduleContext.Builder()
-            .withClientInfo(info)
-            .withStudyIdentifier("test-study")
-            .withUserDataGroups(Sets.newHashSet("parkinson","test_user"))
-                .withEndsOn(DateTime.now().plusDays(1).withHourOfDay(23).withMinuteOfHour(59).withSecondOfMinute(59))
-                .withTimeZone(DateTimeZone.UTC)
-            .withHealthCode("AAA")
-            .withUserId(USER_ID)
-            .withAccountCreatedOn(DateTime.now().minusDays(4))
-            .build();
-        
-        // Is a parkinson patient, gets 3 tasks (or 6 tasks late in the day, see BRIDGE-1603
-        List<ScheduledActivity> schActivities = service.getScheduledActivities(context);
+                    "'minAppVersion':36,"+
+                    "'type':'SchedulePlan'"+
+                "}");
+                
+                Map<String,DateTime> events = Maps.newHashMap();
+                events.put("enrollment", DateTime.now().withZone(DateTimeZone.UTC).minusDays(3));
+                when(activityEventService.getActivityEventMap("AAA")).thenReturn(events);
+                
+                ClientInfo info = ClientInfo.fromUserAgentCache("Parkinson-QA/36 (iPhone 5S; iPhone OS/9.2.1) BridgeSDK/7");
+                
+                SchedulePlan voiceActivityPlan = BridgeObjectMapper.get().readValue(json, SchedulePlan.class);
+                List<SchedulePlan> schedulePlans = Lists.newArrayList(voiceActivityPlan);
+                when(schedulePlanService.getSchedulePlans(info, new StudyIdentifierImpl("test-study"))).thenReturn(schedulePlans);
+                
+                ScheduleContext context = new ScheduleContext.Builder()
+                    .withClientInfo(info)
+                    .withStudyIdentifier("test-study")
+                    .withUserDataGroups(Sets.newHashSet("parkinson","test_user"))
+                        .withEndsOn(DateTime.now().plusDays(1).withHourOfDay(23).withMinuteOfHour(59).withSecondOfMinute(59))
+                        .withTimeZone(timeZone)
+                    .withHealthCode("AAA")
+                    .withUserId(USER_ID)
+                    .withAccountCreatedOn(DateTime.now().minusDays(4))
```

Comment:
Rewrote this test to use a fixed NOW value (this was also fixed in @before with DateTimeUtils.setCurrentMillisFixed() but this is not very easy to follow; actually forcing now to be supplied to ScheduleContext, rather than using DateTime.now() in the build method, might also help reduce time errors in these tests).